install.php

```diff
@@ -138,6 +142,47 @@
 }
 Template::render_header(_("Install"));
 ?>
+<h1 class="text-center"><?php echo _("Prerequisite");?></h1>
+<summary><?php echo _("If any of the following prerequisites are shown as failed (red X), please correct the issue and reload the page before proceeding with the installation.");?></summary>
+<?php
+
+
+	$php_version_req = sprintf(_("Minimum PHP version %s"), MINIMUM_PHP_VERSION);
+    $preq_fail = array("remove", "danger");
+	$preq_ok   = array("ok", "success");
+
+	$preq_phpver = $preq_fail;
+	$preq_mysqlnd = $preq_fail;
+	$preq_writedir = $preq_fail;
+
+	// Check if PHP version if > MINIMUM_PHP_VERSION
+	if (strnatcmp(phpversion(), MINIMUM_PHP_VERSION) >= 0) { $preq_phpver = $preq_ok; }
+
+	// Check if we have mysqlnd installed
+	if ( function_exists('mysqli_get_client_stats') ) { $preq_mysqlnd = $preq_ok; }
```

Comment:
Is this enough to check for mysqlnd? Any specific reason for checking this function? If so please add comment with explanation

Reply:
I believe it is sufficient method to establish that mysqlnd is loaded. The mysqlnd driver provides some extra functions that is not available if the plain mysql package is installed, and `mysqli_get_client_stats` is one of them. This is documented on the PHP site at http://www.php.net/manual/en/mysqlnd.stats.php.

I also verified this on a Centos 7 install by removing the mysqlnd package and installing the plain mysql. With that package the test will fail. 

Another reference I found was this on stackoverflow - https://stackoverflow.com/questions/1475701/how-to-know-if-mysqlnd-is-the-active-driver

Reply:
Comment added to install.php